pkg/imgpkg/cmd/file_flags.go

```diff
@@ -10,11 +10,14 @@ import (
 type FileFlags struct {
 	Files []string
 
-	FileExcludeDefaults []string
+	ExcludedFileBasenames []string
 }
 
 func (s *FileFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().StringSliceVarP(&s.Files, "file", "f", nil, "Set file (format: /tmp/foo, -) (can be specified multiple times)")
 
-	cmd.Flags().StringSliceVar(&s.FileExcludeDefaults, "file-exclude-defaults", []string{".git"}, "Excluded file paths by default (can be specified multiple times)")
+	cmd.Flags().StringSliceVar(&s.ExcludedFileBasenames, "file-exclude-defaults", []string{".git"}, "Excluded file paths by default (can be specified multiple times)")
+	cmd.Flags().MarkDeprecated("file-exclude-defaults", "use '--file-base-exclude' instead")
+
+	cmd.Flags().StringSliceVar(&s.ExcludedFileBasenames, "file-base-exclude", []string{".git"}, "Exclude all files whose base name matches (format: bar.yaml) (can be specified multiple times)")
```

Comment:
I believe that this help should inform the user that this is a relative path, something like:
`Exclude all files whose base name matches (format: bar.yaml) relative to the bundle root (can be specified multiple times)`

Reply:
"base name" typically refers to filename without the path components (eg https://golang.org/pkg/path/filepath/#Base).